src/main/java/de/craften/ui/swingmaterial/MaterialPopupMenu

```diff
@@ -0,0 +1,46 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package de.craften.ui.swingmaterial;
+
+import de.craften.ui.swingmaterial.ElevationEffect;
+import de.craften.ui.swingmaterial.MaterialShadow;
+import java.awt.Color;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import javax.swing.JPopupMenu;
+import javax.swing.border.EmptyBorder;
+
+/**
+ *
```

Comment:
A short description of what this class does and a link to the corresponding Material-Design Spec page is missing here. See the other classes for reference. :wink: